src/component/body/Body.scss

```diff
@@ -13,11 +13,3 @@
   display: flex;
   justify-content: space-around;
 }
```

Comment:
here forgot to separate the classes from each other
Plus you have this part repeated:
```
  display:flex;
   justify-content: space around;
```
Create one reusable class